test/test_language_scanner.py

```diff
@@ -39,7 +39,6 @@ class TestLanguageScanner:
         "paste",
         "phf",
         "quote",
-        "rand",
```

Comment:
What is this change? Is it a bad rebase/merge?

Reply:
The number of Rust packages detected by the tool appeasrs to be changing.